packages/lastrev-adapter-contentful/src/entryParser/entryParser.ts

```diff
@@ -39,7 +39,12 @@ export default (
       _contentTypeId,
       _href,
       _as,
-      _modifiedDate
+      _modifiedDate,
+      id: _id,
```

Comment:
These fields are added to keep parity between parsing a key from a content item and from the load key. The load key is meant to be a partial representation of the content.

Reply:
That makes sense. But still seems to me that the keys should use the _ format, to differentiate them from regular fields. since those fields are not part of the actual content, they are not really partial representations of the content anyway.